samples/smart/README.md

```diff
@@ -1,33 +1,60 @@
-# Azure SMART on FHIR ONC (g)(10) Sample
+# Azure ONC (g)(10) & SMART on FHIR Sample
 
-Sample for ONC (g)(10) / SMART on FHIR work. Still under active development.
+This sample demonstrates how Azure Health Data Services can be used to pass the Inferno test for ONC (g)(10) compliance, using Azure Active Directory as the identity provider. While the FHIR Server is the core of this sample, some custom code and routing is required to fully meet the requirements. This sample is therefore *not* using only the FHIR Server but other Azure Services to pass the Inferno tests.
```

Comment:
Reword to Samples provided in this repository, can be utilized to integrate with Azure Healthcare APIs to comply with requirements under §170.315(g)(10) Standardized API for patient and population services. This sample is tested with integration of Azure Healthcare API's, Azure Active Directory and Azure API Management. This integration has 100% Inferno pass rate for §170.315(g)(10) Standardized API for patient and population services.